### DISCUSSION TITLE
Banded Rows in Saved Search Results and Sublists

### DISCUSSION THREAD

**Discussion by atrent0815** [burkybang/SuiteAdvanced-Feedback#90] [Ideas]
I would love the option to adjust the color scheme and/or contrast for every other row on sublists and saved search results so you can more easily distinguish between each row

**Comment by eduardRsult** [comment]
Yeah, me to! The most hard part would be if it also worked for the lines of eg the item sublist in edit mode. 
Since NS updates the ui when you edit a line, this is pretty challenging.

There are allready alternating colours, however on most screens the contrast is way to low.

Other related features are freeze (mutliple) (left/right) column(s) or top row(s)
